review_app_status.py

```diff
@@ -0,0 +1,137 @@
+# -*- encoding: utf-8
+
+import json
+import logging
+import os
+import time
+from enum import Enum
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("review_app_status")
+
+
+class BuildStates(Enum):
+    """Review App Build states"""
+
+    success = "success"
+
+
+def _make_github_api_request(url):
+    """Perform Github API Queries.
+
+    Input:
+        url: URL to fetch.
+    Output:
+        JSON Response.
+    """
+    headers = {
+        "Accept": "application/vnd.github.v3+json",
+        "Authorization": f"token {os.environ['GITHUB_TOKEN']}",
+    }
+
+    r = requests.get(url, headers=headers)
+    if r.status_code != requests.codes.ok:
```

Comment:
This line is completely redundant. `raise_for_status()` does exactly that.

Reply:
Done